File: c++-programming/chapter-002/precedence.cpp
#include <iostream>

using namespace std;

int main()
{
	int num = 1 + 4 * 3;
	cout << endl << "Default order: " << num << endl;

	num = (1 + 4) * 3;
	cout << "Forced order: " << num << endl << endl;

	num = 7 - 4 + 2;
	cout << "Default direction: " << num << endl;

	num = 7 - (4 + 2);
	cout << "Forced direction: " << num << endl;

	return 0;
}

File: python/chapter-002/precedence.py
a = 2
b = 4
c = 8

print('\nDefault Order:\t', a, '*', c, '+', b, '=', a * c + b)
print('Forced Order:\t', a, '* (', c, '+', b, ') =', a * (c + b))

print('\nDefault Order:\t', c, '//', b, '-', a, '=', c // b - a)
print('Forced Order:\t', c, '// (', b, '-', a, ') =', c // (b - a))

print('\nDefault Order:\t', c, '%', a, '+', b, '=', c % a + b)
print('Forced Order:\t', c, '% (', a, '+', b, ') =', c % (a + b))

print('\nDefault Order:\t', c, '**', a, '+', b, '=', c ** a + b)
print('Forced Order:\t', c, '** (', a, '+', b, ') =', c ** (a + b))
